openid-federation-1_0.xml

```diff
@@ -4601,79 +4598,164 @@ trust_mark=eyJ0eXAiOiJ0cnVzdC1tYXJrK2p3dCIsImFsZyI6 ...
-                        <t hangText="revoked">
-                            <vspace/>
-                            The trust mark is revoked
-                        </t>
-                        <t hangText="invalid">
-                            <vspace/>
-                            Signature validation fails or instance has expired
-                        </t>
-                    </list>
-                </t>
-            </list>
-          </t>
+                REQUIRED. String.
+                Entity Identifier of the issuer of the Trust Mark Status JWT.
+              </t>
+              <t hangText="sub">
+                <vspace/>
+                REQUIRED. String.
+                The Trust Mark JWT that this status response is about.
+              </t>
+              <t hangText="iat">
+                <vspace/>
+                REQUIRED. Number. Time when this Trust Mark Status JWT was issued.
+                This is expressed as Seconds Since the Epoch, per
+                <xref target="RFC7519"/>.
+              </t>
+
+              <t hangText="status">
+                <vspace/>
+                REQUIRED. Case-sensitive string value indicating the status of the Trust Mark.
+		Values defined by this specification are:
+
+		<list style="hanging">
+		  <t hangText="active">
+		    <vspace/>
+		    The Trust Mark is active
+		  </t>
+		  <t hangText="expired">
+		    <vspace/>
+		    The Trust Mark has expired
+		  </t>
+		  <t hangText="revoked">
+		    <vspace/>
+		    The Trust Mark was revoked
+		  </t>
+		  <t hangText="invalid">
+		    <vspace/>
+		    Signature validation failed or another error was detected
+		  </t>
+		</list>
+	      </t>
+	      <t>
+		Additional status values MAY be defined and used
+		in addition to those above.
+	      </t>
+	    </list>
+	  </t>
+	  <t>
+	    Additional Trust Mark Status claims MAY be defined and used
+	    in addition to the one above.
+	  </t>
 
-          <t>
-            An error response
-            is as defined in
-            <xref target="error_response"/>.
-          </t>
-            <t>
-                If the Trust Mark Issuer receives a request about the status of an unknown Trust Mark, something it did not issue or is not aware of,
-                it MUST respond with an HTTP status code 404 (Not found)
-            </t>
           <t>
             <figure>
               <preamble>
-                The following is a non-normative example of a Trust Mark positive status response:
+                The following is a non-normative example of the JWT Claims Set for
+		a response with the status <spanx style="verb">active</spanx>:
               </preamble>
 	      <name>
-		Trust Mark Status Response
+		Active Trust Mark Status Response
 	      </name>
               <artwork><![CDATA[
 200 OK
 Content-Type: application/json
 
 {
-  "active": true
+  "iss": "https://www.agid.gov.it",
+  "sub": "eyJ0eXAiOiJ0cnVzdC1tYXJrK2p3dCIsImFsZyI6IlJTMjU2Iiwia2lkIjoia
```

Comment:
having as subject a JWT containing another subject is not good

we find that the JWT sub is set with `https://rp.example.it/spid`

the sub value must therefore be set with `https://rp.example.it/spid` and a parameter called `trust_mark` should contain, optionally the entire trust mark